nf_core/create.py

```diff
@@ -162,6 +162,11 @@ def create_param_dict(self, name, description, author, version, template_yaml_pa
         param_dict["logo_dark"] = f"{param_dict['name_noslash']}_logo_dark.png"
         param_dict["version"] = version
 
+        # Check that the pipeline name matches the requirements
+        if not re.match(r"^[a-z]+$", param_dict["short_name"]):
+            log.error("[red]Invalid workflow name: must be lowercase without punctuation.")
```

Comment:
```suggestion
            log.error("[red]Invalid name: must be lowercase without punctuation.")
```

Reply:
I will maintain `Workflow name` and instead change the parameter in a different PR as discussed [here](https://github.com/nf-core/tools/pull/1822#issuecomment-1252275795)